args.go

```diff
@@ -36,6 +36,8 @@ const (
 	ArgActionType = "action-type"
 	// ArgApp is the app ID.
 	ArgApp = "app"
+	// ArgWithProjects will determine whether project ids should be fetched along with listed apps.
+	ArgWithProjects = "with-projects"
```

Comment:
The rest of these app-specific args pseudo-namespaced by containing the word "App" - could we do that here as well for consistency?
```suggestion
	ArgAppWithProjects = "with-projects"
```

Reply:
Oh thanks, I missed that bit of namespacing! Will udpate.